File: Parte2/ReservaHotelFront/src/app/models/food.ts
export interface Food {
    id:number;
    name:string;
    price:number;
    tags?:string[];
    // favorite:boolean;
    // stars:number;
    imageUrl:string;
    origins:string[];
    cookTime:string;
}

export interface FoodCreate {
    name:string;
    price:number;
    tags?:string[];
    // favorite:boolean;
    // stars:number;
    imageUrl:string;
    origins:string[];
    cookTime:string;
}

File: Parte2/ReservaHotelBack/ReservaHotel/Entities/Food.cs
namespace ReservaHotel.Entities
{
    public class Food
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public float Price { get; set; }
        public string[] Tags { get; set; }
        public string ImageUrl { get; set; }
        public string[] Origins { get; set; }
        public string CookTime { get; set; }
    }
}
